libqtile/window.py

```diff
@@ -715,13 +715,14 @@ def __init__(self, win, qtile, screen,
         self.conf_y = y
         self.conf_width = width
         self.conf_height = height
-        self.x = x or 0
-        self.y = y or 0
+        x = x or 0
+        y = y or 0
+        self.x = x + screen.cmd_info()['x']
+        self.y = y + screen.cmd_info()['y']
```

Comment:
Can't we get the screen position in a simpler way?

Reply:
Of course
```suggestion
        self.y = y + screen.y
```
Also maybe to get rid of this checks for None
```
        self.x = x or 0
        self.y = y or 0
```
we should change default values of x and y in line 711 to
`                 x=0, y=0, width=None, height=None):`

Reply:
Yeah that sounds good

Reply:
After this change is made, I can merge it and try again to make a test. IMO a squashed bug without a test is better than a present bug.